types/chrome/index.d.ts

```diff
@@ -4896,17 +4896,21 @@ declare namespace chrome.idle {
      * @since Chrome 25
      */
     export function queryState(detectionIntervalInSeconds: number, callback: (newState: IdleState) => void): void;
+    export function queryState(detectionIntervalInSeconds: number): Promise<IdleState>;
```

Comment:
- [ ] If I remember correctly, the promised version should be on top
- [ ] This method lacks a JSDoc block, it should be copy-pasted from above (updating the `@since` prop to match the now in the docs)

Reply:
I have made the changes for this in the new commit.